### PR TITLE
CollisionDefine for Swift

### PR DESCRIPTION
**Pull request by normancroan**
Added CollisionDefine.m and modified CollisionDefine.h to support Swift
implementations.

### CODE REVIEW

**Comment by benkane** [comment]
Can we do the README update and the collision define update as 2 separate pull requests since they are unrelated?

**Comment by normancroan** [comment]
Yes, reverted README update in this PR.

File: SKAToolKitExample/SKATiledMap/CollisionDefine.h [hunk 1]
```diff
@@ -8,9 +8,10 @@
 
 #ifndef SKATMXParser_CollisionDefine_h
 #define SKATMXParser_CollisionDefine_h
+#import <Foundation/Foundation.h>
```
Comment (benkane) [inline SKAToolKitExample/SKATiledMap/CollisionDefine.h:11]: @normancroan Is this import necessary?

Comment (normancroan) [inline SKAToolKitExample/SKATiledMap/CollisionDefine.h:11]: Yes, it does not work without it

Comment (benkane) [inline SKAToolKitExample/SKATiledMap/CollisionDefine.h:11]: Ok

**Comment by benkane** [comment]
Looks good. Can you rebase this down to one commit?

**Comment by normancroan** [comment]
Yeah I'll give that a try now

**Comment by benkane** [comment]
:+1: Looks good, merging

**Pull request merged**